docs/guides/trivy.md

```diff
@@ -0,0 +1,55 @@
+# Moving towards configuration scanning with Trivy
+Overtime we've taken [trivy][trivy] to be the go-to scanning tool for a vareity of things. This also includes terraform scanning.
+
+This section describes the differences between Trivy and tfsec.
+
+| Feature               | Trivy                                                  | tfsec                |
+|-----------------------|--------------------------------------------------------|----------------------|
+| Built-in Policies     | :material-check:                                       | :material-check:     |
+| Custom Policies       | Rego                                                   | Rego, JSON, and YAML |
+| Policy Metadata[^1]   | :material-check:                                       | :material-check:     |
```

Comment:
not sure what this was supposed to mean actually. I don't think this is any differentiator, wdyt?

Reply:
yeah agree - I updated the table to only include the differences